src/en/discover/technology/index.html

```diff
@@ -38,7 +38,7 @@ <h2 class="color-white h1">Ceph delivers object, block, and file storage in a si
 <section class="section">
   <div class="max-w-192 mx-auto text-center">
     <h2 class="h2">The Ceph Stack -- Architectural Overview</h2>
-    <a href="#object"><img alt="" src="/assets/stack.png" /></a>
+    <a href="#object"><img alt="" src="assets/bitmaps/stack.png" /></a>
```

Comment:
Thanks for adding this asset, @zdover23.

Just one minor note on the `src` paths. Because we won't always necessarily know at what depth the page that's referencing the image is, it's safest to always use root-relative paths:

```suggestion
    <a href="#object"><img alt="" src="/assets/bitmaps/stack.png" /></a>
```

That will ensure the content is as predictable and portable as possible. (Exception perhaps with the blog posts, where local images are very tightly coupled with their content and references.)